src/Core.UnitTests/Notifications/NotificationServiceTests.cs

```diff
@@ -454,56 +434,13 @@ public void ShowNotification_InfoBarButtonClicked_NonCriticalException_Exception
             logger.AssertPartialOutputStringExists("this is a test");
         }
 
-        [TestMethod]
```

Comment:
This test is still valid, but for CloseNotification. Why completely removing it and not adapt it instead?

Reply:
It's a duplicated test